packages/scxa-cell-type-wheel-experiment-heatmap/src/CellTypeWheelExperimentHeatmap.js

```diff
@@ -51,13 +51,20 @@ function CellTypeWheelExperimentHeatmap (props) {
       />
       <div className={`row-expanded small-12 columns`}>
         <div className={`small-12 medium-6 columns`} aria-label={`Cell type wheel`}>
-          <CellTypeWheelFetchLoader
-            host={props.host}
-            resource={URI(props.searchTerm, props.cellTypeWheelResource).toString()}
-            fulfilledPayloadProvider={ cellTypeWheelData => ({ data: cellTypeWheelData }) }
-            searchTerm={props.searchTerm}
-            onCellTypeWheelClick={onCellTypeWheelClick}
-          />
+          {props.searchTerm.length > 0 ?
+            <CellTypeWheelFetchLoader
+              host={props.host}
+              resource={URI(props.searchTerm, props.cellTypeWheelResource).toString()}
+              fulfilledPayloadProvider={cellTypeWheelData => ({data: cellTypeWheelData})}
+              searchTerm={props.searchTerm}
+              onCellTypeWheelClick={onCellTypeWheelClick}
+            /> :
+            <div className={`medium-text-center`} aria-label={`Empty search term`}>
+              <h4>
+                Please enter a search term to be able to see the cell type wheel.
```

Comment:
Cell type wheel is kind of our team term, I would use a more neutral/biologically oriented phrase, e.g.: “Please type a term in the search box to see a distribution of cell types across species”. I always double-check with curators, though.

Reply:
Changed